Standards/scs-xxxx-v1-provider-network-standard.md

```diff
@@ -0,0 +1,213 @@
+---
+title: SCS Provider Network Standard
+type: Standard
+status: Proposal
+track: IaaS
+---
+
+## Introduction
+
+Many use-cases of IaaS require virtual servers to be able to connect to network resources outside of the cloud environment, often to the internet.
+Openstack supports this by allowing CSPs to map non-virtualized networks onto specific virtual networks, such that virtual routers and servers can connect to them.
+
+Such networks will usually be created in a provider-managed project and then shared to user projects using the RBAC-feature of the network API.
+Because they have to be set up by the cloud provider, networks of this type are generally referred to as _provider networks_, though that term can also be used in a broader senseto refer to other types of CSP-managed virtual networks.
```

Comment:
```suggestion
Because they have to be set up by the cloud provider, networks of this type are generally referred to as _provider networks_, though that term can also be used in a broader sense to refer to other types of CSP-managed virtual networks.
```